CGATPipelines/pipeline_enrichment/pipeline.ini

```diff
@@ -22,6 +22,73 @@ version=0.1
 
 # The full version, including alpha/beta/rc tags.
 release=0.1
+
+
+
+############################################################
+# Options for running each enrichment analysis
+############################################################
+[analysis]
+# Analysis type, options are:
+# GSEA (Geneset enrichment analysis) and
+# ORA  (Over Representative Analysis)
+gsea=1
+ora=1
+
+##########################################################
+# Parameters for GSEA analysis.
+
+
+[id_gsea]
+# id type of input gene list.
+type=entrez
+# which gene id type is used for the analysis
+# can be ensemblg, entrez or symbol
+# The analysis will be based on this ID type 
+# for symbol, append the organism id of the species.
+# e.g. symbol_Homo_sapiens, symbol_Mus_musculus
+## organism id in entrez - 9606 = human, 10090 = mouse
+# others here: http://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/
+# NOTE:id type of gene list and gene set should be same. This option is 
+# only for translating gene list id to your required id type.
+to_convert=entrez
+
+[geneset]
+# path to geneset
+name=/ifs/projects/proj073/GSEA_PIPELINE/Gene_set.gmt
+
+[db_gsea]
+# path to a annotated database generated using pipeline_geneinfo.py
+name=/ifs/projects/reshma/DATABASE/human_db_110817
```

Comment:
Is `human_db_110817` a file that should be located in a project independent folder?